pkg/plugins/runtime/k8s/controllers/cni_taint_controller.go

```diff
@@ -187,18 +193,6 @@ func podEvents(cniApp string) predicate.Funcs {
 	}
 }
 
-func filterNodes(obj kube_client.Object) bool {
-	node, ok := obj.(*kube_core.Node)
-	if !ok {
-		return false
-	}
-	if node.Name == "" {
-		return false
-	}
-
-	return true
-}
-
 func filterPods(obj kube_client.Object, cniApp string) bool {
```

Comment:
Could remove the `NodeName` check from this function since it's checked in the mapper.

Reply:
Done, you were right [here](https://github.com/kumahq/kuma/pull/4650#discussion_r937655346)

Reply:
done